src/contrib/cluster/Akka.Cluster.Sharding/Serialization/ClusterShardingMessageSerializer.cs

```diff
@@ -69,9 +77,9 @@ public ClusterShardingMessageSerializer(ExtendedActorSystem system) : base(syste
         {
             _fromBinaryMap = new Dictionary<string, Func<byte[], object>>
             {
-                {EntityStateManifest, EntityStateFromBinary},
-                {EntityStartedManifest, EntityStartedFromBinary},
-                {EntityStoppedManifest, EntityStoppedFromBinary},
+                {EntityStateManifest, bytes => EntityStateFromBinary(bytes) },
```

Comment:
I prefer using method straight away instead of lambda syntax - I don't know it lambda syntax will trigger generation of hidden class representing it, but this way I'm sure I don't give a compiler reasons to think it should try so.

Reply:
The existing serializer does a mix of both already and these messages aren't sent very often.